### PR TITLE
Fix build instructions for MSVC

### PR DESCRIPTION
**Pull request by monote**
Fixed 'change directory' commands and names of certain files to be correctly qualified when running build process step-by-step.

### CODE REVIEW

File: docs/building-msvc.md [hunk 1]
```diff
@@ -134,7 +135,8 @@ Open **x86 Native Tools Command Prompt for VS 2017.bat**, go to ***BuildPath***
     git checkout v5.6.2
     cd ..\qtbase
     git checkout v5.6.2
-    git apply ../../../tdesktop/Telegram/Patches/qtbase_5_6_2.diff
+    git apply ../../../../tdesktop/Telegram/Patches/qtbase_5_6_2.diff
```
Comment (stek29) [inline docs/building-msvc.md]: I think this line was correct

Comment (monote) [reply docs/building-msvc.md]: ...! You are correct. Revising and will resubmit.

**Comment by monote** [comment]
30 minutes later, this issue finally fixed. The patience is appreciated. Just getting used to GitHub.

**Comment by monote** [comment]
Re-doing pull request entirely. Just realized nasty syntax on the new submission. No reason to have three commits for something so small.

**Pull request closed without merging**